portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -9,10 +9,31 @@
 /** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
 @WebServlet("/hello")
 public class HelloWorldServlet extends HttpServlet {
+    String[] quotes = {"You know more than you think you know, just as you know less than you want to know.", 
+                        "One day I will find the right words, and they all will be simple.",
+                        "I assure you that the world is not so amusing as something we imagined."};
+
+    String json = convertToJson(quotes);
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+  }
+
+  /**
+   * Converts a ServerStats instance into a JSON string using manual String concatentation.
```

Comment:
Since this is specific to quotes, it's probably better to change the comment description to reference that, or make this method more general (e.g. changing the quotes parameter name to something more general).